tests/integration/cassettes/TestUserApiSdk.test_insert.yaml

```diff
@@ -0,0 +1,315 @@
+      X-Request-Id:
+      - 76035711-c879-46c3-8659-70e034a23ecc
+      X-Runtime:
+      - '0.064434'
+      X-XSS-Protection:
+      - 1; mode=block
+    http_version: HTTP/1.1
+    status_code: 200
+- request:
+    body: ''
+    headers:
+      accept:
+      - '*/*'
+      accept-encoding:
+      - gzip, deflate
+      connection:
+      - keep-alive
+      user-agent:
+      - python-httpx/0.22.0
+    method: DELETE
+    uri: null
+  response:
+    content: '{"deleted_count":0}'
+    headers:
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '19'
+      Content-Type:
+      - application/json
+      Date:
+      - Wed, 18 May 2022 13:10:08 GMT
+      server:
+      - uvicorn
+    http_version: HTTP/1.1
+    status_code: 200
+- request:
+    body: ''
+    headers:
+      accept:
+      - '*/*'
+      accept-encoding:
+      - gzip, deflate
+      connection:
+      - keep-alive
+      user-agent:
+      - python-httpx/0.22.0
+    method: GET
+    uri: null
+  response:
+    content: '[]'
+    headers:
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '2'
+      Content-Type:
+      - application/json
+      Date:
+      - Wed, 18 May 2022 13:10:09 GMT
+      server:
+      - uvicorn
+    http_version: HTTP/1.1
+    status_code: 200
+- request:
+    body: '[{"asset_id": 83776921, "version": 1, "data": {"k": "v"}, "timestamp":
+      10}, {"asset_id": 83776921, "version": 1, "data": {"k": "v"}, "timestamp": 11}]'
+    headers:
+      accept:
+      - '*/*'
+      accept-encoding:
+      - gzip, deflate
+      connection:
+      - keep-alive
+      content-length:
+      - '150'
+      content-type:
+      - application/json
+      user-agent:
+      - python-httpx/0.22.0
+      x-corva-app:
+      - python-sdk-autotest-2022-05-18 13:10:06+00:00
+    method: POST
+    uri: null
+  response:
+    content: '{"inserted_ids":["6284f0323cc8096aa7e5c8f8","6284f0323cc8096aa7e5c8f9"],"failed_count":0,"messages":[]}'
+    headers:
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '103'
+      Content-Type:
+      - application/json
+      Date:
+      - Wed, 18 May 2022 13:10:10 GMT
+      server:
+      - uvicorn
+    http_version: HTTP/1.1
+    status_code: 200
+- request:
+    body: ''
+    headers:
+      accept:
+      - '*/*'
+      accept-encoding:
+      - gzip, deflate
+      connection:
+      - keep-alive
+      user-agent:
+      - python-httpx/0.22.0
+    method: GET
+    uri: null
+  response:
+    content: '[{"_id":"6284f0323cc8096aa7e5c8f8","company_id":80,"asset_id":83776921,"version":1,"provider":"big-data-energy","collection":"corva-sdk-autotest","data":{"k":"v"},"timestamp":10,"app_key":"big-data-energy.oleksii_test_task_app"},{"_id":"6284f0323cc8096aa7e5c8f9","company_id":80,"asset_id":83776921,"version":1,"provider":"big-data-energy","collection":"corva-sdk-autotest","data":{"k":"v"},"timestamp":11,"app_key":"big-data-energy.oleksii_test_task_app"}]'
```

Comment:
Should we give `big-data-energy.oleksii_test_task_app` a more generic name? Like `big-data-energy.python_sdk_test_task_app`?

Reply:
Done, thanks